web/src/main/webapp/libs/app.bundle.min.js

```diff
@@ -117117,11 +117186,15 @@
             const {
                 _calendarEnabled: e,
                 _recentListEnabled: t,
+                _inProgressListEnabled: u, // BAO
                 t: n
             } = this.props, r = [];
             return e && r.push({
                 label: n("welcomepage.calendar"),
                 content: v.a.createElement(Na.b, null)
+            }), u && r.push({ // BAO
+                label: n("welcomepage.inProgressList", {ns: "ofmeet"}),
+                content: v.a.createElement(InProgresstList, null)
```

Comment:
Add the above InProgressList component to the tab when InProgressList is enabled.